### ISSUE TITLE
Bundler update

### ISSUE THREAD

**Issue by jochenjonc** [BretFisher/jekyll-serve#80]
Hi Bret,

Are you planning to push an updated image with the lastest or newer bundler release? I use the image for local development and it always creates bundles with version 2.3.22.
But for deployment, I build with Oryx via GitHub Actions and it always first tries to use a more recent version of bundler (2.4.2).

It would be nice if locally it would use a higher version aswel.

Regards,

Jochen

**Comment by BretFisher** [comment]
Hmm, `gem install bundler` still installs 2.3.26.

**Comment by jochenjonc** [comment]
Could it be because of the ruby or rubygems version?

bundler 2.4.x requires ruby >= 2.6.0 or higher and rubygems >= 3.0.1
bundler 2.3.x requires ruby >= 2.3.0 or higher and rubygems >= 2.5.2

**Comment by BretFisher** [comment]
OK, this should be fixed with PR #27 that just landed:

```text
docker run -it --entrypoint bash bretfisher/jekyll:latest
root@583f8d8bf59a:/site# gem --version
3.4.1
root@583f8d8bf59a:/site# ruby --version
ruby 3.2.0 (2022-12-25 revision a528908271) [aarch64-linux]
root@583f8d8bf59a:/site# bundler --version
Bundler version 2.4.3
```

I'm closing the issue for now, and thanks for the report.
